### PR TITLE
Added former 2 committees

### PR DESCRIPTION
**Pull request by ash-xyz**
Added the members of the last 2 committees. 
Let's not forget history :smile:

### CODE REVIEW

**Comment by oisinq** [comment]
Good idea. I think this could be better on its own page?

**Comment by ash-xyz** [comment]
Added a new page called history with all known former NetSoc committee members

**Comment by ash-xyz** [comment]
Added a new page called history with all known former NetSoc committee members

File: history.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
+---
+layout: page
+title: History
+permalink: /about/history/
+---
+Originally founded in 1995 to provide UCD students with Internet access, we now run tech-focused events.
+
+### Committee 2019/20:
+* Auditor: Thomas Creavin
+* Vice Auditor: Nicola McCabe
```
Comment (oisinq) [inline history.md]: This should be Nicole (did i do it wrong in the old website? oops)

**Review by oisinq — CHANGES_REQUESTED** [review]
One small change

**Comment by oisinq** [comment]
@all-contributors please add @ash-xyz for content

**Pull request merged**